### PR TITLE
PR on noverbal simon, translated modules, ultimate cipher condensed and interactive

### PR DESCRIPTION
**Pull request by Limeboy0603**
All work:

non verbal simon pdf reupload
espik's nonverbal simon manual update
added czech translation of colored switched, the screw

notify me in #maintainence if there are any problems. thx

### CODE REVIEW

File: HTML/Colored Switches translated (Čeština — Barevných Spínačů).html [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+    <link rel="stylesheet" type="text/css" href="css/normalize.css">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+    <script src='js/ktane-utils.js'></script>
+</head>
+<body>
+<div class="section">
+    <div class="page page-bg-02">
+        <div class="page-header">
+            <span class="page-header-doc-title">Keep Talking and Nobody Explodes Mod</span>
+            <span class="page-header-section-title">Barevné Spínače (čeština) v1.2</span>
```
Comment (red031000) [inline HTML/Colored Switches translated (Čeština — Barevných Spínačů).html]: please use the title from this line

File: HTML/Colored Switches translated (Čeština — Barevných Spínačů).html [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+        </div>
+        <div class="page-content">
+            <img src="img/Component/Colored Switches.svg" class="diagram">
+            <h2>Na téma Barevných Spínačů (čeština)</h2>
+            <p class="flavour-text">Je to pouze pět spínačů. Jak těžký by to mohlo být?
+            <ul>
+                <li>Modul obsahuje pět spínačů. Každý spínač může být oranžový, zelený, fialový, tyrkysový, modrý nebo červený.
```
Comment (red031000) [inline HTML/Colored Switches translated (Čeština — Barevných Spínačů).html]: please close these tags

File: HTML/The Screw translated (Čeština — Šroubu).html [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+            line-height: 105%;
+        }
+    </style>
+</head>
+<body>
+    <div class="section">
+        <div class="page page-bg-02">
+            <div class="page-header">
+                <span class="page-header-doc-title">Keep Talking and Nobody Explodes Mod</span>
+                <span class="page-header-section-title">Šroub (čeština) v1.2</span>
```
Comment (red031000) [inline HTML/The Screw translated (Čeština — Šroubu).html]: again, please take the title from this line

**Comment by Limeboy0603** [comment]
maybe frank should really check on the titles

**Comment by Limeboy0603** [comment]
do not merge yet i havnt finish the pdf rename

**Comment by Limeboy0603** [comment]
finally done

File: HTML/Colored Switches translated (Čeština — Barevné Spínače).html [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+        <div class="page-header">
+            <span class="page-header-doc-title">Keep Talking and Nobody Explodes Mod</span>
+            <span class="page-header-section-title">Barevné Spínače (čeština) v1.2</span>
+        </div>
+        <div class="page-content">
+            <img src="img/Component/Colored Switches.svg" class="diagram">
+            <h2>Na téma Barevné Spínače (čeština)</h2>
+            <p class="flavour-text">Je to pouze pět spínačů. Jak těžký by to mohlo být?</p>
+            <ul>
+                <li>Modul obsahuje pět spínačů. Každý spínač může být oranžový, zelený, fialový, tyrkysový, modrý nebo červený.
```
Comment (red031000) [inline HTML/Colored Switches translated (Čeština — Barevné Spínače).html]: please close these <li>s before starting the next one

File: HTML/Ultimate Cipher condensed (Fang).html [hunk 5]
```diff
@@ -0,0 +1,1861 @@
...
+			border: black 2px solid;
+			transition: 0.3s;
+		}
+
+		button:hover {
+			background-color: black;
+			color: white;
+		}
+    </style>
+    <script src="js/jquery.3.1.1.min.js"></script>
```
Comment (red031000) [inline HTML/Ultimate Cipher condensed (Fang).html]: if this is interactive, please mark this as such

Comment (Limeboy0603) [reply HTML/Ultimate Cipher condensed (Fang).html]: he said condensed thou, so i did both interactive and condensed, it is pretty much condensed lol

**Pull request closed without merging**